### ISSUE TITLE
Hierarchy of values is missing

### ISSUE THREAD

**Issue by LigaSaleniece** [statisticssweden/PxWeb#496]
**Is your feature request related to a problem? Please describe.**

After the implementation of WCAG in PX-Web interface, the value hierarchy feature disappeared.
Unfortunately, hierarchical meaning cannot always be ensured by displaying the value code before the value, as it does not contain information that can be interpreted in this way.

**Describe the solution you'd like**
AT CNMM already has the ability to use grouping and choose a hierarchical view. But this is not provided through the PX-Web interface. It would be great if such an opportunity could be provided.
